validator/test/fixtures/assets/projects/data.json

```diff
@@ -0,0 +1,74 @@
+[{
```

Comment:
Can we rename the directory (`project`) after the constraint template name? The expectation is that we don't plan to share test fixtures across templates to minimize any cross-test dependency on fixture data.

Reply:
Reorganized the tests according to the discussion in #63